### ISSUE TITLE
Documentation missing: PolarGrid

### ISSUE THREAD

**Issue by nikolasrieble** [recharts/recharts#3400]
The properties stroke, stroke-dasharray, stroke-width, etc. are not documented on the PolarGrid.
(Unfortunately they are not documented in many places 😓 )

https://twitter.com/pomodorozhong/status/1628129110504656899

<img width="605" alt="Screenshot 2023-02-26 at 08 10 13" src="https://user-images.githubusercontent.com/27995606/221397256-fa6fd172-027e-4eed-88dd-a8563e3dca41.png">

**Comment by ckifer** [comment]
The reason those properties aren't documented is because they're SVGProps which recharts has historically not documented. 

I still go back and forth on what we should document and what we should not. I'm of the opinion that not all SVGProps should be blindly accepted by recharts and instead we should document everything we do accept very explicitly and refine chart props to reflect that.

**Comment by nikolasrieble** [comment]
I agree, so lets take this as a chance to document what we do want to support. 
stroke, stroke-width, and stroke-dashArray seem to be both useful and reasonable to support. 

Presumably a whitelist would have to be on component level, right? Or does it make sense to accept, i.e. stroke / fill everywhere and start with a global whitelist (which then is to be documented, and to be used for filtering in an upcoming release)

**Comment by ckifer** [comment]
I remember talking briefly about the (ESM only) library that gave us correct SVG attributes for the element types. Doing this would be breaking so adding that on 3.0 would be reasonable. From there the only issue becomes getting correct types for each element type. 

For now making the documentation even to each component that does accept the common props is probably the best we can do.
